src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionId.java

```diff
@@ -35,6 +35,8 @@ public abstract class ModuleExtensionId {
               ModuleExtensionId::getIsolationKey,
               emptiesFirst(IsolationKey.LEXICOGRAPHIC_COMPARATOR));
 
+  public static final String INNATE_EXTENSION_NAME = "__innate";
```

Comment:
Assuming that this string is going to be visible to users in lockfiles and via `bazel mod`, I would prefer a less cryptic name. What do you think of `_repository_rules` (a single leading underscore should suffice)?

Reply:
+1, due to my limited vocabulary, I had to look up this word to understand its meaning 😅

Reply:
Me too :-)

Reply:
changed!